oc/oc-netobserv-flows-db

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+source "./scripts/functions.sh"
+
+# interface filter such as 'br-ex'
+filter=""
+
+if [ -z "${1:-}" ]
+then
+  echo "Filters not set"
+else
+  echo "Filters set as $1"
+  filter=$1
+fi
+
+# CLI image to use
+img="quay.io/mmahmoud/network-observability-cli:main-amd64"
+
+trap cleanup EXIT
+
+setup flows $filter
+
+echo "Running network-observability-cli get-flows-db... "
+oc run \
+  -n netobserv-cli \
+  collector \
+  --image=$img\
+  --image-pull-policy='Always' \
+  --restart='Never' \
+  --command -- sleep infinity
+
+oc wait \
+  -n netobserv-cli \
+  --for=condition=Ready pod/collector
+
+oc exec -i --tty \
+  -n netobserv-cli \
+  collector \
+  -- /network-observability-cli get-flows-db ${filter:+"--filter" "$filter"}
```

Comment:
We should improve the arguments of `oc get flows` command allowing `filters` and / or `output format` instead of creating a new script here.

I made the disctinction between `flows` / `packets` / `cleanup` commands only to make these appear in the oc plugin list

Reply:
If your idea was also to get rid of the display that consume resources, that could be an extra argument :nerd_face:

Reply:
I merged both in the same command and removed the db one also kept the console out

Reply:
thanks ! That's awesome